x-pack/plugin/src/test/resources/rest-api-spec/test/ml/data_frame_analytics_crud.yml

```diff
@@ -2100,3 +2100,53 @@ setup:
               "includes": ["excluded"]
             }
           }
+
+---
+"Test update given all updatable settings":
+
+  - do:
+      ml.put_data_frame_analytics:
+        id: "update-test-job"
+        body: >
+          {
+            "source": {
+              "index": "index-source"
+            },
+            "dest": {
+              "index": "index-dest"
+            },
+            "analysis": {"outlier_detection":{}},
+            "description": "before update",
+            "model_memory_limit": "20mb",
+            "allow_lazy_start": false
+          }
+  - match: { id: "update-test-job" }
+  - match: { description: "before update" }
+  - match: { model_memory_limit: "20mb" }
+  - match: { allow_lazy_start: false }
+
+  - do:
+      ml.update_data_frame_analytics:
+        id: "update-test-job"
+        body: >
+          {
+            "description": "after update",
+            "model_memory_limit": "30mb",
+            "allow_lazy_start": true
+          }
+  - match: { id: "update-test-job" }
+  - match: { description: "after update" }
+  - match: { model_memory_limit: "30mb" }
+  - match: { allow_lazy_start: true }
+
+---
+"Test update given missing analytics":
+
+  - do:
+      catch: missing
+      ml.update_data_frame_analytics:
```

Comment:
I don't see a `update_data_frame_analytics.json` definition file.

Reply:
Me neither.
Apart from that, the test looks good!

Reply:
Yeah, I never added it :-(